doc/source/examples/sharp-immersed-boundary/flow-around-particle-sharp/flow-around-particle-sharp.rst

```diff
@@ -0,0 +1,229 @@
+compared to the expected results from the literature. 
+
+.. image:: images/flow-around-particle-sharp-case.png
+    :alt: Simulation schematic
+    :align: center
+
+---------------
+Parameter File
+---------------
+
+Mesh
+~~~~
+
+As shown in the previous figure, the flow around the particle is modelled using a prismatic channel the size of which depends on the size of the particle. In this case, we consider a particle with a diameter of :math:`D = 1`, therefore we have :math:`L = 2`. 
+The domain, of size :math:`32 L \times 16 L \times 16 L`, was found to produce results independant of domain sizes with a low computational cost. 
+
+The mesh is first defined coarsely away from the particle with elements of size :math:`L \times L \times L`. The particle is positionned at :math:`3/8` of the domain length to avoid boundary interactions and allow the flow to develop both behind and in front of the particle.
+We then make use of a ``box refinement`` to refine the mesh around the particle and properly evaluate the velocity field with elements of size :math:`1/4 D \times 1/4 D \times 1/4 D`.
+
+.. code-block:: text 
+
+    subsection mesh
+        set type                   = dealii
+            set grid type          = subdivided_hyper_rectangle
+            set grid arguments     = 2,1,1 : -24,-16,-16 : 40,16,16 : true
+            set initial refinement = 4
+        end
+
+    subsection box refinement 
+        set initial refinement = 3
+        subsection mesh
+            set type               = dealii
+            set grid type          = subdivided_hyper_rectangle
+            set grid arguments     = 2,1,1: -6,-4,-4 : 10,4,4 : true
+            set initial refinement = 0
+        end
+    end
+
+Immersed Boundary Particles
+~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+We define the particle in flow using the sharp immersed boundary method. The particle is defined as a cylinder with a radius of :math:`D/2 = 0.5` and a half-length of :math:`L/2 = 1.0`. The particle is centered at :math:`(0,0,0)` 
+and the cylider's axis is oriented perpendicularly to direction of the flow. 
+
+The ``local mesh refinement`` is set to 3 to ensure we resolve the boundary layer flow around the particle with elements of size :math:`1/32 D \times 1/32 D \times 1/32 D`.
+
+.. code-block:: text
+
+    subsection particles
+        set number of particles                     = 1
+        set assemble Navier-Stokes inside particles = false
+        subsection extrapolation function
+            set stencil order = 2
+            set length ratio  = 1
+        end
+        subsection local mesh refinement
+            set initial refinement                = 3
+            set refine mesh inside radius factor  = 0.75
+            set refine mesh outside radius factor = 2
+        end
+        subsection particle info 0
+            set type            = cylinder
+            set shape arguments = 0.5;1.0
+            set integrate motion = false
```

Comment:
```suggestion
```

This is not in the .prm file anymore

Reply:
Good point, I'll add it to the prm as it should be there!